python/cuml/test/test_meta_estimators.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.pipeline import Pipeline
+from cuml.model_selection import GridSearchCV
+
+from cuml.datasets import make_classification
+from cuml.model_selection import train_test_split
+from sklearn.datasets import load_iris
+
+from cuml.experimental.preprocessing import StandardScaler
+from cuml.svm import SVC
+
+
+def test_pipeline():
```

Comment:
Should we test that every model is compatible? seems like we could do this with about 10 rows of data and make it fast. Could use the same approach as test_pickle to grab at least all regression and classification models pretty easily. This is an open question, so LMK if you think it's a bad idea...
 It might be nice also to confirm that output type doesn't get messed up. Basically, predict should preserve the on-gpu-ness of the data even through the pipeline (predict + check output class type). Not sure the scoring is even necessary since this is more a question of whether the pipeline is flowing...